build.gradle

```diff
@@ -14,7 +14,7 @@
  */
 
 buildscript {
-    ext.kotlin_version = '1.1.4-3'
+    ext.kotlin_version = '1.1.51'
```

Comment:
Let's bumb to 1.2.0 so that the browser and the sdk are using the same kotlin version.